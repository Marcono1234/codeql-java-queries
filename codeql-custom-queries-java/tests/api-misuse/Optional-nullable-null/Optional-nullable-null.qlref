./api-misuse/Optional-nullable-null.ql
