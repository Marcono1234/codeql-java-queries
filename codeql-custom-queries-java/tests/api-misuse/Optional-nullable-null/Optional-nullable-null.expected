| Test.java:6:9:6:33 | ofNullable(...) | Should use Optional.empty() |
